packages/terra-paginator/src/_paginationUtils.jsx

```diff
@@ -1,3 +1,61 @@
+import * as KeyCode from 'keycode-js';
+
+/**
+ * Enables focus styles for the target of the given event. Typically used as an onBlur callback on selectable elements.
+ * See: terra-application-navigation/src/utils/helpers.js
+ */
+const enableFocusActiveStyles = (event) => {
+  if (event.nativeEvent.keyCode === KeyCode.KEY_SPACE) {
+    event.currentTarget.setAttribute('data-active-styles-enabled', 'true');
+  }
+
+  if (event.nativeEvent.keyCode === KeyCode.KEY_SPACE || event.nativeEvent.keyCode === KeyCode.KEY_RETURN) {
+    event.currentTarget.setAttribute('data-focus-styles-enabled', 'true');
+  }
+};
+
+/**
+ * Disables focus styles for the target of the given event. Typically used as an onMouseDown callback on selectable elements.
+ * See: terra-application-navigation/src/utils/helpers.js
+ */
+const disableFocusActiveStyles = (event) => {
+  if (event.nativeEvent.keyCode === KeyCode.KEY_SPACE) {
+    event.currentTarget.setAttribute('data-active-styles-enabled', 'false');
+  }
+
+  if (event.nativeEvent.keyCode === KeyCode.KEY_TAB) {
+    event.currentTarget.setAttribute('data-focus-styles-enabled', 'true');
+  }
+};
+
+/**
+ * Disable focus style on Blur event.
+ */
+const disableFocusOnBlur = (event) => {
+  event.currentTarget.setAttribute('data-focus-styles-enabled', 'false');
+};
+
+let shouldShowFocus = true;
```

Comment:
Why is this being defined here? Should this be managed in the component's state?

Reply:
I went ahead and added this to the button's state, and passed the setter to the utility functions.